docs/installation/windows.mdx

```diff
@@ -12,119 +12,34 @@ import Admonition from '@theme/Admonition';
-          - We support older processors with AVX and AVX-512, though this is not recommended.
-        </Admonition>
-        - Excavator processors (Q2 2015) and newer.
-      </TabItem>
-    </Tabs>
-  </TabItem>
-  <TabItem value="RAM" label="RAM">
-    - 8GB for running up to 3B models (int4).
-    - 16GB for running up to 7B models (int4).
-    - 32GB for running up to 13B models (int4).
-      <Admonition type="info">  
-        We support DDR2 RAM as the minimum requirement but recommend using newer generations of RAM for improved performance.
-      </Admonition>
-  </TabItem>
-  <TabItem value="GPU" label="GPU">
-    - 6GB can load the 3B model (int4) with `ngl` at 120 ~ full speed on CPU/ GPU.
-    - 8GB can load the 7B model (int4) with `ngl` at 120 ~ full speed on CPU/ GPU.
-    - 12GB can load the 13B model (int4) with `ngl` at 120 ~ full speed on CPU/ GPU.
-    <Admonition type="info">  
-    - [NVIDIA driver](https://www.nvidia.com/Download/index.aspx) version 470.63.01 or higher.
-    - [CUDA Toolkit](https://developer.nvidia.com/cuda-toolkit) version 12.3 or higher.
-    </Admonition>
-  </TabItem>
-  <TabItem value="Disk" label="Disk">
-    - At least 10GB for app storage and model download.
-  </TabItem>
-</Tabs>
-
-## Cortex Installation
-
-### Step 1: Install Cortex
+## Overview
+For Windows, Cortex can be installed either directly on **Windows** or on **Windows Subsystem for Linux (WSL)**.
 
 :::info
-You can also install Cortex using the Cortex Installer available on [GitHub Releases](https://github.com/janhq/cortex/releases).
+If you're unsure about what is **Windows Subsystem for Linux (WSL)**, please refer to this [article](https://learn.microsoft.com/en-us/windows/wsl/faq).
 :::
 
+## Windows
+Before installation, make sure that you have met the required [dependencies](#windows) and [hardware](#hardware) to run Cortex.
 ```sh
-# Install globally on your system
+# Install using NPM
 npm i -g cortexso
 ```
-:::info
-Cortex automatically detects your CPU and GPU, downloading the appropriate CPU instruction sets and required dependencies to optimize GPU performance.
-:::
-
-### Step 2: Verify the Installation
-
-1. After installation, you can verify that Cortex is installed correctly by getting help information.
-
-```sh
-# Get the help information
-cortex -h
-```
-2. Cortex is ready to use!
 
 ## Windows Subsystem Linux
-
-### Step 1: Open your WSL2 Terminal
-
-Open your Linux terminal in WSL2. For WSL2, you can use the Linux distribution terminal, which is Ubuntu.
-
-### Step 2: Install Cortex
-
-:::info
-You can also install Cortex using the Cortex Installer available on [GitHub Releases](https://github.com/janhq/cortex/releases).
-:::
+Before installation, make sure that you have met the required [dependencies](#windows-subsystem-for-linux) and [hardware](#hardware) to run Cortex.
 
 ```sh
-# Install globally on your system
+# Install using NPM
 npm i -g cortexso
 
 # Install using Brew
 brew install cortexso
```

Comment:
Cmiiw, brew cant be used for windows (Im coming from Mac Land), so lets remove brew? I also dont see Brew in the original notion for windows
https://www.notion.so/jan-ai/Windows-b85b40faefbb40eea2f81f2716f4cc9a?pvs=4